src/polycubed/src/extiface.cpp

```diff
@@ -254,5 +255,119 @@ std::string ExtIface::get_iface_name() const {
   return iface_;
 }
 
+/* Interfaces alignment */
+void ExtIface::subscribe_event(const std::string &param_name,
+      std::function<void(const std::string, const std::string)> callback) {
+  if (param_name == "MAC_ADDRESS") {
```

Comment:
I defined the strings and changed the name from "IP_ADDREES" and "MAC_ADDRESS" to "IP" and "MAC".
In case of changes, it is sufficient to modify the definition.